app/controllers/telegram/webhook_controller.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+class Telegram::WebhookController < Telegram::Bot::UpdatesController
+  def message(message)
+    chat_id = message['chat']['id']
+    user = User.find_by(chat_id: chat_id)
+    unless user
```

Comment:
> Once you've send an initial message to our bot, you will be registered as a user.

I’m not sure that’s the behavior we want, the audience for the tool should be closed, i.e. users shouldn’t be able to signup theirselves. If I understood correctly, it would be more like this:

1. An editor or administrator imports a list of users. Eventually, I guess there should be a UI for it, but for now (and for this PR), I think it would be enough if we manually set them up in the DB.
2. If users would like to communicate via Telegram, they need to either
  a) use the [login with Telegram](https://telegram.org/blog/login) button or
  b) send an initial message to the bot (as it is implemented right now).
  (As 100eyes should be transparent to the end user – at least for now – I think I’d prefer b).

cc @FrauCsu

Reply:
You can get information about the user like this:

```ruby
message.user.username
message.user.first_name
message.user.last_name
```

So in case we want to manually add users using their Telegram username, we could initially use the username to match an incoming message to a user in a database, then save `message.chat_id` and use it for all subsequent incoming messages (to account for a changed username).